docs/blog/newsletter-april-2022.rst

```diff
@@ -0,0 +1,137 @@
+
+.. post:: April 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – April 2022
+#########################################
+
+Hey everyone! It's Beth and the newsletter team, back with another edition full of news and views from the docs world.
+
+2022 is a big year for Write the Docs: it will be our 10th year of conferences in North America. To celebrate, we've added a video of an O'Reilly Interview of Eric and Marcia Riefer Johnston from 2015 to our "Origin Story" page. `Watch and enjoy </origin-story/#where-we-are-now>`__!
+
+And if you fancy joining us for that 10th NA conference, you might like to know that we've just announced our `fabulous speakers </conf/portland/2022/news/announcing-speakers/>`__. Lots to get excited about there!
+
+To finish, a bit of news from your editor. After four lovely years and 35 (!!) issues, it's time for me to hand over stewardship of the newsletter to someone new. This is such a great opportunity to give back to the WTD community; so if you think it might be for you, take a look at `this blog post about the role </blog/newsletter-editor/>`__ and get in touch.
```

Comment:
```suggestion
To finish, a bit of news from your editor. After four lovely years and 35 (!!) issues, it's time for me to hand over stewardship of the newsletter to someone new. This is such a great opportunity to give back to the WTD community; so if you think it might be for you, take a look at our blog post about the role, `Looking for a new newsletter editor </blog/newsletter-editor/>`__ and get in touch.
```

Reply:
I wasn't sure if the original construction breaks the "'this' in links" rule, but here's an alternative idea

Reply:
Much better. Sorry, I'm terrible at figuring out how to avoid this!